modules/ipi-manual-migration-to-customdeploy.adoc

```diff
@@ -0,0 +1,45 @@
+// This is included in the following assemblies:
+//
+// ipi-install-post-installation-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="manual-migration-to-new-customdeploy-install-method_{context}"]
+
+= Manual migration to new customDeploy install method
+
+A new deployment method introduced in 4.10 allows you to customize the network configuration (`networkConfig`) in the `install-config.yaml` file by host during the install and provisioning process. You can also set static IPs per host and additional advanced network configurations.
+
+When you upgrade to {product-title} 4.10, the new deployment method is not automatically upgraded and therefore you need to perform the following manual steps.
```

Comment:
I'm wondering if we want to explain that although most functionalities would not affected, this change is necessary before trying to scale up the cluster.